tests/functional/sign_up.js

```diff
@@ -289,41 +318,163 @@ define([
         .end();
     },
 
-    'coppa does not allow sign up if younger than 13 years old': function () {
-      return FunctionalHelpers.fillOutSignUp(this, email, PASSWORD, { age: 12 })
+    'sign up with existing account, coppa is valid, credentials are correct': function () {
+      return signUpWithExistingAccount(this, email, PASSWORD, PASSWORD)
 
-        // Success is being redirected to the cannot create screen.
-        .findById('fxa-cannot-create-account-header')
+        // should have navigated to settings view
+        .findByCssSelector('#fxa-settings-header')
         .end();
     },
 
+    'sign up with existing account, coppa is valid, credentials are wrong': function () {
+      return signUpWithExistingAccount(this, email, PASSWORD, 'bad' + PASSWORD)
+
+        // should have navigated to sign-in view
+        .findByCssSelector('#fxa-signin-header')
+        .end()
+
+        // should be /signin route
+        .getCurrentUrl()
+          .then(function (url) {
+            assert.ok(url.indexOf('/signin') > -1);
+            assert.equal(url.indexOf('/oauth/signin'), -1);
+          })
+        .end()
+
+        // an error should be visible
+        .then(FunctionalHelpers.testErrorWasShown(this))
+
+        // the email field should be populated
+        .findByCssSelector('input[type=email]')
```

Comment:
We should make a helper to check the value of an element, so we could do something like:

```js
.then(testElementValueEquals('input[type=email'], email))
```